### PR TITLE
3.0.1 Mender release version updates

### PR DESCRIPTION
**Pull request by oleorhagen**
Changelog: None
Signed-off-by: Ole Petter <ole.orhagen@northern.tech>

### CODE REVIEW

**Comment by oleorhagen** [comment]
Note: Links are still untested (in fact the binaries are not even there yet), so don't merge

File: 05.System-updates-Yocto-Project/05.Customize-Mender/01.Delta-update-support/docs.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ command:
 <!--AUTOVERSION: "mender-binary-delta/%/mender-binary-delta-%.tar"/mender-binary-delta-->
 ```bash
 HOSTED_MENDER_EMAIL="myusername@example.com"
-curl -u $HOSTED_MENDER_EMAIL -O https://downloads.customer.mender.io/hosted/content/mender-binary-delta/1.2.1/mender-binary-delta-1.2.1.tar.xz
+curl -u $HOSTED_MENDER_EMAIL -O https://downloads.customer.mender.io/hosted/content/mender-binary-delta/1.3.0/mender-binary-delta-1.3.0.tar.xz
```
Comment (lluiscampos) [inline 05.System-updates-Yocto-Project/05.Customize-Mender/01.Delta-update-support/docs.md:29]: We shouldn't change the minor version of mender-binary-delta on a bugfix release.

**Review by lluiscampos — CHANGES_REQUESTED** [review]
See comment below wrt mender-binary-delta.

Also, what about mender-convert images? We can sync on this on Slack but we need to publish and update the links.

File: 05.System-updates-Yocto-Project/05.Customize-Mender/01.Delta-update-support/docs.md (hunk 1 above)
Comment (oleorhagen) [reply 05.System-updates-Yocto-Project/05.Customize-Mender/01.Delta-update-support/docs.md:29]: Roger, slight misunderstanding between me and Kristian yesterday

**Comment by oleorhagen** [comment]
> See comment below wrt mender-binary-delta.
> 
> Also, what about mender-convert images? We can sync on this on Slack but we need to publish and update the links.

I'm doing convert separately, if that is okay?

**Review by lluiscampos — APPROVED** [review]

**Pull request merged**